automation/all_clippy_checks.sh

```diff
@@ -18,7 +18,7 @@ fi
 EXTRA_ARGS=( "$@" )
 
 # Later rust versions downgraded some warning to pedantic, so we allow them here.
-ALLOWS=("-Aclippy::unreadable-literal"  "-Aclippy::trivially-copy-pass-by-ref" "-Aclippy::match-bool" "-Aunknown-lints")
+ALLOWS=("-Aclippy::unreadable-literal"  "-Aclippy::trivially-copy-pass-by-ref" "-Aclippy::match-bool" "-Aunknown-lints" "-Aclippy::unknown_clippy_lints")
```

Comment:
TIL that "unknown lints" are not the same thing as "unknown clippy lints" and given the way we pin our Rust version, I really don't think they should block CI.

Reply:
LOL, for posterity: https://rust-lang.github.io/rust-clippy/master/#unknown_clippy_lints

Reply:
Hahahaha wow; looking forward to getting on an updated Rust here at some point!